news/2024/2024-09-30-the-followpoint-d-i-o-the-mastermind-behind-osu-tournaments.md

```diff
@@ -0,0 +1,227 @@
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/11354436" style="background-image: url('/wiki/shared/news/2024-09-30-the-followpoint-d-i-o-the-mastermind-behind-osu-tournaments/avatar-Utiba.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/11354436">Utiba</a></p>
+
+As a host of The Perennial tournament series, what makes it stand out to you that makes you want to continue hosting the tournament each year?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3958619" style="background-image: url('/wiki/shared/news/2024-09-30-the-followpoint-d-i-o-the-mastermind-behind-osu-tournaments/avatar-D I O.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/3958619">D I O</a></p>
+
+When I first started it, the Perennial was one of two well-regarded large community 4v4 tournaments in osu! standard. At the start, I focused on a large prize pool and community hype to draw in as many people as possible. Now, several years later, my focus has shifted more to uniqueness in prizing and format while still preserving a similar level of production as previous iterations.
+ 
+The Perennial has never been completely unique, so it instead stands out with the competitiveness of its matches, especially in late stages. In 2021, this was not so much the case, but in 2022 and 2024, we had very competitive grand finals matches between the top teams, with both those grand finals going to bracket reset. This was partially due to the format change in 2024. Watching these competitive, high-level matches in a tournament you host, on a pool you made, is always very rewarding.
+
+With that in mind, the Perennial has become an opportunity for me to attempt to influence the tournament meta on a larger scale. I often use smaller tournaments to test run ideas I have for the Perennial before implementing them there, which was a big success this year with regional team restrictions. My hope is that these modifications keep the tournament as competitive as possible and continue to keep it at the forefront of people's minds when thinking about tournament formats.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/11354436" style="background-image: url('/wiki/shared/news/2024-09-30-the-followpoint-d-i-o-the-mastermind-behind-osu-tournaments/avatar-Utiba.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/11354436">Utiba</a></p>
+
+You have casted for some of the biggest tournaments in osu!. Did you ever foresee yourself becoming such a well known and respected figure in the community because of this? Are there any professional casters you looked up to during this journey who helped you find your style?
```

Comment:
```suggestion
You have casted for some of the biggest tournaments in osu!. Did you ever foresee yourself becoming such a well-known and respected figure in the community because of this? Are there any professional casters you looked up to during this journey who helped you find your style?
```